tooling/src/hypothesistooling/__main__.py

```diff
@@ -399,6 +399,7 @@ def check_py27():
 
 @python_tests
 def check_py34():
+    exit(1)
```

Comment:
This is breaking the Travis build - instead of exiting here, you can just omit `TASK: check-py34` from the Azure config.

Reply:
Aw beans, this line was added in anger to break the azure pipelines faster when iterating and then somehow forgot about it. I'll fix that real quick; thanks for catching it!